data/Inputs/BatchFiles/European_SingleORAL_Age_0_CYP3A4.json

```diff
@@ -187,7 +187,7 @@
             },
             {
               "Name": "Resolution",
-              "Value": 4.0,
+              "Value": 0.5707,
```

Comment:
this will result in exactly 5000 output points as before, so that batch comparison should return valid for all simulations now :)

Reply:
Lol..we should support number of points directly lol